### PR TITLE
[RFR] Added OSP Image count checks.

### PR DESCRIPTION
**Pull request by sbulage**
- [X]  Added Image count check.

### CODE REVIEW

File: check-osp/osp_checks.py [hunk 1]
```diff
@@ -38,6 +38,27 @@ def check_vm_count(system, warn=10, crit=15, **kwargs):
     sys.exit(3)
 
 
+def check_image_count(system, warn=10, crit=15, **kwargs):
+    """ Check overall Image count. """
+    warn = int(warn)
+    crit = int(crit)
+    template_count = len(system.list_templates())
```
Comment (kedark3) [inline check-osp/osp_checks.py]: Let's call it `image_count` ?

File: check-osp/osp_checks.py [hunk 2]
```diff
@@ -38,6 +38,27 @@ def check_vm_count(system, warn=10, crit=15, **kwargs):
...
+    # determine ok, warning, critical, unknown state
+    if template_count < warn:
```
Comment (kedark3) [inline check-osp/osp_checks.py]: same here

File: check-osp/osp_checks.py [hunk 3]
```diff
@@ -38,6 +38,27 @@ def check_vm_count(system, warn=10, crit=15, **kwargs):
...
+        print("Ok: Image count is less than {}. Image Count = {}".format(warn, template_count))
+        sys.exit(0)
+    elif template_count >= warn and template_count <= crit:
+        print("Warning: Image count is >=  {} & <= {}. Image Count = {}"
+            .format(warn, crit, template_count))
+        sys.exit(1)
+    elif template_count > crit:
+        print("Critical: Image count is greate than crit. Image Count = {}".format(crit, template_count))
```
Comment (kedark3) [inline check-osp/osp_checks.py]: One `{}` missing here. should be 
`print("Critical: Image count is greater than critical {}. Image Count = {}".format(crit, template_count))`

**Review by kedark3 — CHANGES_REQUESTED** [review]
Thanks for PR. Please fix some things.

File: check-osp/osp_checks.py (hunk 3 above)
Comment (sbulage) [reply check-osp/osp_checks.py]: Oops :smile:

**Comment by sbulage** [comment]
Updated.

**Review by kedark3 — APPROVED** [review]

**Pull request merged**